Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -159,6 +159,14 @@ The hosting settings allow you to host buckets in the object store on a custom D
 
 * `dnsNames`: a list of DNS names to host buckets on. These names need to valid according RFC-1123. Otherwise it will fail. Each endpoint requires wildcard support like [ingress loadbalancer](https://kubernetes.io/docs/concepts/services-networking/ingress/#hostname-wildcards). Do not include the wildcard itself in the list of hostnames (e.g., use "mystore.example.com" instead of "*.mystore.example.com"). Add all the hostnames like openshift routes otherwise access will be denied, but if the hostname does not support wild card then virtual host style won't work those hostname. By default cephobjectstore service endpoint and custom endpoints from cephobjectzone is included. The feature is supported only for Ceph v18 and later versions.
 
+!!! Important considerations:
+
+* Domains mentioned here should listen to same port as RGW kubernetes service.
+
+* If RGW kubernetes service listen to both `secure` and `non-secure` port, include domains which uses secure port.
```

Comment:
To be included in the "important" block, these lines must be indented four spaces, and I'm not sure it supports bullets for proper rendering. @BlaineEXE do you know? Perhaps we just make a paragraph instead of bullets.
```suggestion
    If the RGW Kubernetes service listens to both `secure` and `non-secure` ports, include domains which use a secure port.
```

Reply:
It won't support bullets inside the callout. This should just be part of the doc.